src/include/stir/Scanner.inl

```diff
@@ -384,7 +463,21 @@ Scanner::get_transaxial_singles_unit(int singles_bin_index) const {
   return(singles_bin_index % get_num_transaxial_singles_units());
 }
 
-
+// For retrieving the coordinates / detector, ring id from the scanner
+stir::DetectionPosition<>
+Scanner::get_detpos_given_id(const stir::DetectionPosition<> det_pos) const{
```

Comment:
making these accept a `const &` could speed it up a tiny-bit

Reply:
done.